depends/packages/native_cxxbridge.mk

```diff
@@ -0,0 +1,16 @@
+package=native_cxxbridge
+$(package)_version=1.0.68
+$(package)_download_path=https://github.com/dtolnay/cxx/archive/refs/tags
+$(package)_file_name=native_cxxbridge-$($(package)_version).tar.gz
+$(package)_download_file=$($(package)_version).tar.gz
+$(package)_sha256_hash=6fed9ef1c64a37c343727368b38c27fa4e15b27ca9924c6672a6a5496080c832
+$(package)_build_subdir=gen/cmd
+$(package)_dependencies=native_rust
+
+define $(package)_build_cmds
+  cargo build --release --package=cxxbridge-cmd --bin=cxxbridge
```

Comment:
Currently this is not fully pinned, because the upstream repo does not include any `Cargo.lock`. I don't think this should affect reproducible builds, because it would require a Rust-SemVer-compatible update to a dependency of `cxxbridge` that affects how C++ code generation occurs, which seems unlikely.

Nevertheless this is probably going to cause Gitian problems, so it would be good to figure out a reasonable solution; probably a "patch" that applies a `Cargo.lock` to the build source (though IDK how nice that will be to keep updated). We also need to solve the vendoring problem since we won't have network access at this point of Gitian to fetch the dependencies in the first place, so we need to vendor during `$(package)_fetch_cmds`.

Reply:
I think I have both the pinning and vendoring implemented and working now.